README.md

```diff
@@ -1,15 +1,24 @@
 # slc -- StrongLoop Controller
 
-`slc` is a command-line tool for development and control of your Node
-application.
+`slc` is the command shell for the StrongLoop Process Manager,
```

Comment:
Yes, that's how I look at it. I'd prefer to rename `slc arc` to `slc gui`, or even just `slc` (the default command would be the GUI).